modules/workflow/src/main/java/org/eclipse/xpanse/modules/workflow/definition/ServiceMigrationWorkflow.java

```diff
@@ -0,0 +1,90 @@
+package org.eclipse.xpanse.modules.workflow.definition;
+
+import static io.nflow.engine.workflow.definition.WorkflowStateType.end;
+
+import io.nflow.engine.workflow.curated.State;
+import io.nflow.engine.workflow.definition.NextAction;
+import io.nflow.engine.workflow.definition.StateExecution;
+import io.nflow.engine.workflow.definition.StateVar;
+import io.nflow.engine.workflow.definition.WorkflowDefinition;
+import io.nflow.engine.workflow.definition.WorkflowState;
+import io.nflow.engine.workflow.definition.WorkflowStateType;
+import java.util.UUID;
+import lombok.extern.slf4j.Slf4j;
+import org.eclipse.xpanse.modules.deployment.DeployService;
+import org.eclipse.xpanse.modules.models.service.deploy.CreateRequest;
+import org.eclipse.xpanse.modules.orchestrator.deployment.DeployTask;
+import org.eclipse.xpanse.modules.orchestrator.deployment.Deployment;
+import org.eclipse.xpanse.modules.workflow.utils.WorkflowUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+/**
+ *
+ */
+@Slf4j
+@Component
+public class ServiceMigrationWorkflow extends WorkflowDefinition {
```

Comment:
since we will have both terraform-boot and terraform-local.. i dont think this workflow will work also for terraform-boot. Better to enable this only for profiles - terraform-local and nflow